cli/src/schemas/WorkingGroups.ts

```diff
@@ -49,7 +49,7 @@ export const WorkingGroupOpeningInputSchema: JsonSchema<WorkingGroupOpeningInput
       required: ['amount', 'unstakingPeriod'],
       properties: {
         amount: { type: 'integer', minimum: 2000 },
-        unstakingPeriod: { type: 'integer', minimum: 43201 },
+        unstakingPeriod: { type: 'integer', minimum: 43200 },
```

Comment:
[api.consts.staking.bondingDuration](https://github.com/Joystream/joystream/blob/master/pioneer/packages/page-staking/src/Actions/useUnbondDuration.ts#L21)

Reply:
Hey,
That only applies to staking for validators! I know how to get these, but they could be different both by group, and opening type (Lead vs Worker). If I were to try, it would take some non-trivial amount of time, and would likely not be very clean :)